site/includes/languagetoggle.hbs

```diff
@@ -1,11 +1,21 @@
-<section id="wb-lng" class="text-right">
+<div id="wb-lng" class="col-xs-3 col-sm-12 text-right">
 	<h2 class="wb-inv">{{{i18n "tmpl-lang-select"}}}</h2>
-	<ul class="list-inline margin-bottom-none">
+	<ul class="list-inline margin-bttm-0">
 	{{#isnt language "fr" }}
-		<li><a lang="fr" hreflang="fr" href="{{altLangPrefix}}-fr.html">{{{i18n "lang-native" language="fr"}}}</a></li>
+		<li>
+			<a lang="fr" hreflang="fr" href="{{altLangPrefix}}-fr.html">
+				<span class="hidden-xs">{{{i18n "lang-native" language="fr"}}}</span>
+				<abbr title="Français" class="visible-xs h3 mrgn-tp-sm mrgn-bttm-0">FR</abbr>
```

Comment:
Use the "FR" as defined by i18n file here: https://github.com/wet-boew/wet-boew/blob/master/src/i18n/i18n.csv
and use jekyll liquid to make it all upper case: https://shopify.github.io/liquid/filters/upcase/

```
{{i18n "lang-code" language="fr" | upcase }}
```

Reply:
I get an error that it is invalid. I think it has to be something like `{{ "STRING" | upcase }}`.

Reply:
`<abbr title="Français" class="visible-xs h3 mrgn-tp-sm mrgn-bttm-0 text-uppercase">{{{i18n "lang-code" language="fr"}}}</abbr>`

Reply:
How about this? @duboisp

Reply:
Yes the text-uppercase CSS work too. Let's use it for now. 
But the abbr element should create a double underline because that is a link which is might not fully aligned with the intended design. @delisma your call.